htcrystalball/examine.py

```diff
@@ -42,11 +42,11 @@ def prepare(cpu: int, gpu: int, ram: str, disk: str, jobs: int,
         If all needed parameters were given
     """
 
-    with open(config_file) as f:
-        config = json.load(f)['slots']
+    with open(config_file) as slot_file:
+        config = json.load(slot_file)['slots']
```

Comment:
*Technically* not necessary, as `f` (or `fh`, or `file_handler`) is commonly used for file handlers and it is only used inside of the `with` section. Personally, I think `open(config_file) as slot_file` is more confusing because the unaware reader may wonder what the difference is between a config- and a slot-file, and why an opened config-file becomes a slot-file. Nitpicky, I know. Sorry.

Reply:
I agree that this is nitpicky, but I agree with Niels' points.